.gitignore

```diff
@@ -14,3 +14,4 @@ coverage
 test/typescript/.idea/*
 test/typescript/*.js
 test/typescript/*.map
+package-lock.json
```

Comment:
Got it, I didn't notice about it too much before. Maybe we could also add a `.npmrc` file with `package-lock = false`

just like what eslint did https://github.com/eslint/eslint/issues/8742